pom.xml

```diff
@@ -225,16 +226,25 @@
             </dependency>
             <!-- In *theory* the log4j12 bridge should not be made convergent, as WE decide which way logs flow.
                  Libraries should *only* rely on the api package. But sometimes... :-/ -->
+            <!-- Replacing the dependency slf4j-log4j12 with slf4j-reload4j, -->
+            <!-- in order to make sure we are getting rid of log4j for good... -->
+            <!-- it may not be necessary, to include slf4j-reload4j explicitly -->
+            <!-- but shouldn't hurt either (? L.A. - Jan. 2022) --> 
             <dependency>
                 <groupId>org.slf4j</groupId>
-                <artifactId>slf4j-log4j12</artifactId>
+                <artifactId>slf4j-reload4j</artifactId>
                 <version>${slf4j.version}</version>
```

Comment:
**suggestion:** It looks like we don't need this anymore - none of our deps is including this as a transitive dependency anymore.

Reply:
That's what it looked like to me too - I was going to double-check with others, then remove. But yes, it appears to be safe to remove...